index.html

```diff
@@ -950,11 +956,253 @@ <h3>
+          or not, are advised to support any government verifier authentication
+          schemes, if they exist. The support for (and integration of) verifier
+          authentication schemes such as <a href=
+          "https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/annexes/annex-2/annex-2-high-level-requirements.md#a2327-topic-27---registration-of-pid-providers-providers-of-qeaas-pub-eaas-and-non-qualified-eaas-and-relying-parties">
+          EUDI access and registration certificates</a> can mitigate risks of
+          proliferation of unnecessary credential requests. However, the
+          presence of such schemes is not guaranteed, which significantly
+          increases the risk in a credential exchange.
+        </p>
+        <p>
+          There are other practical steps that user agents implementing the
+          Digital Credentials API can take to reduce risk, increase user
+          understanding and prevent certain types of harm.
+        </p>
+        <ul>
+          <li>Requiring protocols to support selective disclosure and other
+          techniques for data minimization can reduce the impact and likelihood
+          of information leakage, and provide better context to users in
+          permission and consent flows.
+          </li>
+          <li>Requiring protocols to support unlinkability mechanisms such as
+          Zero-Knowledge Proofs can prevent verifier-based surveillance, and
+          potential discrimination by hiding the issuer.
+          </li>
+          <li>Offering useful context and a clearly understandable permission
+          flow will help users make better decisions on whether or not to
+          accept a credential exchange, which can reduce the viability of
+          exchange requests that are made without a concrete user need.
+          </li>
+        </ul>
+        <p>
+          It is further critical that user agents design a permission
+          experience that accounts for the lack of these mitigations, e.g. the
+          exchange of personal information from government credentials without
+          any verifier authentication scheme. It is recommended to apply a very
+          high level of friction for these types of high risk exchanges.
+        </p>
+        <h4>
+          Non-government-issued credentials
+        </h4>
+        <p>
+          Non-government-issued credentials include all other digital
+          documents, certificates and attestations that are not
+          government-issued and don't represent government-issued documents.
+          This could include proof of employment, education credentials or
+          cinema tickets. Notably, their exchange is likely less restricted
+          through laws and regulations. While these documents often don't
+          exhibit the same risks as government-issued credentials, they could
+          also contain identifiable or sensitive information.
```

Comment:
```suggestion
          Non-government-issued credentials include all other digital
          documents, certificates, and attestations that are not
          government-issued and don't represent government-issued documents.
          This could include proof of employment, education credentials, or
          cinema tickets. Notably, their exchange is likely less restricted
          by laws and regulations. While these documents often don't exhibit
          the same risks as government-issued credentials, they could also
          contain identifiable or sensitive information.
```